_infra/helm/reporting/templates/deployment.yaml

```diff
@@ -155,6 +155,20 @@ spec:
           - name: S3_BUCKET
             value: "{{ .Values.aws.bucket }}"
           {{- end }}
+          - name: REPORTING_URL
+            {{- if .Values.ingress.enabled }}
+            value: "https://{{ .Values.ingress.gcpHost }}"
+            {{- else }}
+            value: "http://localhost:{{.Values.service.port}}"
+            {{- end }}
+          - name: DASHBOARD_URL
```

Comment:
I don't quite understand why you've added these here?

I think both the REPORTING URL and DASHBOARD_URL can be removed, as this app doesn't need to know about the dashboard url and the reporting url is this app itself.